### PR TITLE
SettingsActivity to permanently set link type (see issue #18)

### PR DESCRIPTION
**Pull request by mahula**

### CODE REVIEW

**Comment by pR0Ps** [comment]
I had a quick glance at the diff and it looks good. Once I find a bit of time I'd like to build it locally and mess around with it before hitting merge. Thanks for taking care of this!

File: app/src/main/res/xml/preferences.xml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<PreferenceScreen xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <ListPreference
+        android:key="@string/list_pref_Key"
```
Comment (pR0Ps) [inline app/src/main/res/xml/preferences.xml]: change to "prefLinkType"

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -22,4 +22,9 @@
     <string name="openstreetmap" translatable="false">OpenStreetMap</string>
     <string name="geouri" translatable="false">GeoURI</string>
 
+    <string name="settings">Settings</string>
+    <string name="list_pref_Key" translatable="false">prefLinkType</string>
```
Comment (pR0Ps) [inline app/src/main/res/values/strings.xml]: This can just be hardcoded in `preferences.xml` (see below)

File: app/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-          xmlns:tools="http://schemas.android.com/tools"
-          package="ca.cmetcalfe.locationshare">
+    xmlns:tools="http://schemas.android.com/tools"
```
Comment (pR0Ps) [inline app/src/main/AndroidManifest.xml]: minor: re-indent

File: app/src/main/AndroidManifest.xml [hunk 4]
```diff
@@ -16,9 +16,17 @@
         <activity android:name=".MainActivity">
             <intent-filter>
                 <action android:name="android.intent.action.MAIN" />
+
```
Comment (pR0Ps) [inline app/src/main/AndroidManifest.xml]: minor: delete extra line

**Comment by mahula** [comment]
Changes have been made.

File: app/src/main/AndroidManifest.xml [hunk 5]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-          xmlns:tools="http://schemas.android.com/tools"
```
Comment (pR0Ps) [inline app/src/main/AndroidManifest.xml]: revert this indentation please

**Comment by pR0Ps** [comment]
Thanks!

**Comment by mahula** [comment]
Before bumping to the next version, increasing "targetSdkVersion" and "buildToolsVersion" to current availlible versions would be good, eg. to make LocationShare available for devices running Android 8.x.

**Comment by pR0Ps** [comment]
[Noted](#21), thanks

**Pull request merged**